app/user/apis.py

```diff
@@ -93,3 +94,215 @@ async def user_edit(
     """This endpoint is used to edit the user's details"""
     user = await services.edit_user(user_id=user.id, data=user_in, db=db)
     return {"data": user}
+
+
+@router.post(
+    "/token",
+    summary="Generate User Access Token",
+    response_description="The user's access token",
+    status_code=status.HTTP_200_OK,
+    response_model=ResponseSchema,
+)
+async def user_refresh_token(
+    db: DatabaseSession,
+    refresh_token: str = Body(
+        description="The user's refresh token", min_length=1, embed=True
+    ),
+):
+    """This endpoint generates a new access token for the user using the refresh token"""
+    if user_id := security.verify_user_refresh_token(token=refresh_token):
```

Comment:
this will always return the id, so no need

Reply:
Okay